src/libraries/Common/src/Interop/Browser/Interop.JavaScript.Function.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    /// <summary>
+    /// The Function constructor creates a new Function object. Calling the constructor directly can create functions dynamically, but suffers from security and similar (but far less significant) performance issues similar to eval. However, unlike eval, the Function constructor allows executing code in the global scope, prompting better programming habits and allowing for more efficient code minification.
+    /// </summary>
```

Comment:
Did you mean for this comment to be on Function rather than on JavaScript?

Reply:
Yes, it goes with Function because the JavaScript function works differently than eval.

Reply:
But does need to be in remarks and have committed a change for that.